lib/instream/connection/config.ex

```diff
@@ -30,6 +30,7 @@ defmodule Instream.Connection.Config do
   def runtime(otp_app, conn, keys)          do
     otp_app
     |> Application.get_env(conn, [])
+    |> Keyword.new
```

Comment:
Apart from missing brackets is this really a necessary step? You defined this as a `bug fix` but from what I could reconstruct it might also be just called a `configuration error`.

The application configuration done using `Mix.Config` does not explicitly require a special format, but the most commonly used format is `Keyword.t`. That means it is possible to have one key defined multiple times with different values:

```
config :app, [ foo: :bar, foo: :baz ]
```

Calling `Keyword.new([ foo: :bar, foo: :baz ])` guarantees the keys are unique and therefore overwrites the first value with the second. So the configuration value for `:foo` would be `:baz`. Your commit wording however said "get first key instead of last" and that seems like the complete opposite of what is actually happening here.

Can you provide a configuration that is problematic without adding this line? If there is a real bug I would extract it and create a patch release for just that fix.